masterlist.yaml

```
@@ -6753,8 +6753,10 @@ plugins:
         util: '[SSEEdit v4.0.1](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 773
 
-  - name: '(Unique Region Names|URN.*).esp'
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/11750/' ]
+  - name: '(Unique Region Names|URN.*)\.esp'
```

I've moved the extended plugins to their own [page](https://www.nexusmods.com/skyrimspecialedition/mods/41853).
The patches for the base file all have `URN_` at the start of their name.
The extended patches all have `URN Extended_`.

```suggestion
  - name: '(Unique Region Names|URN_.*)\.esp'
```